scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -4202,7 +4202,17 @@ class ScalametaParser(input: Input)(implicit dialect: Dialect) { parser =>
     }
   }
 
-  def blockStatSeq(): List[Stat] = listBy[Stat] { stats =>
+  def blockStatSeq(allowRepeated: Boolean = false): List[Stat] = listBy[Stat] { buf =>
+    object stats {
+      final def +=(elem: Stat): Unit = {
+        if (allowRepeated)
+          buf.lastOption match { // only last one can be repeated
+            case Some(t: Term.Repeated) => syntaxError("repeated argument not allowed here", at = t)
+            case _ =>
+          }
```

Comment:
It seems that in case of repeated in braces compiler allow only a single statement.
The following won't compile:
```
scala> def bar(a: Int*): Unit = println(s"ooo ${a}")
def bar(a: Int*): Unit

scala> bar {
     |   val a = List(1,2,3)
     |   a: _*
     | }
         a: _*
          ^
On line 4: error: no `: _*` annotation allowed here
```

Reply:
done, PTAL